Not kill -9 neo4j when stopping it, but log a warning instead

davidegrohmann opened the pull request:
In such a way it is possible to give the process some extra time to
complete and it is also helpful in case debugging is necessary.

It is always possible to manuall kill the process.

tinwelint commented on packaging/standalone/src/main/distribution/shell-scripts/bin/neo4j:
```diff
@@ -215,38 +213,38 @@ startit() {
-	      ## wait for start, pick up the server listening on the port
-	      detectrunning
-	      if [ $newpid ] ; then
-	         break
```
Pretty please try to avoid this amount of formatting changes in scripts, because as it's hard to read changes through simultaneous formatting change in Java code (which we are familiar with), it's even harder for scripts which feels much more uncomfortable making changes in.

davidegrohmann replied:
Alright, I'll revert it

tinwelint commented on packaging/standalone/src/main/distribution/shell-scripts/bin/utils:
```diff
@@ -234,34 +234,22 @@ stopit() {
+      if [ $x -ge $TIMEOUT ] ;then
+	  echo ""
+	  echo "$FRIENDLY_NAME [$NEO4J_PID] is taking more than ${TIMEOUT}s to stop"
+	  echo "Stopping $FRIENDLY_NAME [$NEO4J_PID]..."
```
This looks strange. We're printing `Stopping neo4j...` _after_ saying `...neo4j is taking more than bla to stop`. Could we instead say something like `leaving the server running so that it can try to stop itself, and you can go in and view logs and stuff` Well, obviously not like that, but something in that fashion?

davidegrohmann replied:
I'll try to improve the message.

davidegrohmann committed:
Not kill -9 neo4j when stopping it, but log a warning instead

In such a way it is possible to give the process some extra time to
complete and it is also helpful in case debugging is necessary.

It is always possible to manuall kill the process.

davidegrohmann commented:
@tinwelint I have improved the warning message, please have an extra look

tinwelint commented:
During acceptance testing I spotted a problem. When the wait actually happens (set it to 20sec in my test and had the db wait a bit longer), I see spamming like this:

```
Stopping Neo4j Server [4172]........................
Neo4j Server [4172] is taking more than 20s to stop
There might be some troubles with the shutdown of Neo4j Server, please read log files for details
This script will keep waiting fot the service to shutdown, you could manually kill the process 4172

.
Neo4j Server [4172] is taking more than 20s to stop
There might be some troubles with the shutdown of Neo4j Server, please read log files for details
This script will keep waiting fot the service to shutdown, you could manually kill the process 4172

.
Neo4j Server [4172] is taking more than 20s to stop
There might be some troubles with the shutdown of Neo4j Server, please read log files for details
This script will keep waiting fot the service to shutdown, you could manually kill the process 4172

.
Neo4j Server [4172] is taking more than 20s to stop
...
...
```

every second. Missing a loop break or something? Also the `period` sign on two lines down looks odd.

davidegrohmann commented:
@tinwelint I forgot the '.', my bad. I though keep printing was a good idea, probably it is not. I'll print the warning message only once then.
